app/views/scans/_form_additional.html.erb

```diff
@@ -1,5 +1,11 @@
-<%= f.text_field :page_range, help: 'Example: 1-5, 8, 7-13' %>
+<div class="form-group">
+  <%= f.label :page_range, class: label_column_class + ' control-label ' %>
```

Comment:
You need to use double quotes to take advantage of string interpolation.

So `"#{label_column_class} control-label"` instead of `'#{label_column_class} control-label'`.